src/alfasim_sdk/result_reader/aggregator.py

```diff
@@ -179,13 +269,13 @@ def from_dict(cls, data: Dict[str, Any]) -> Self:
                 qoi_data_index=data["qoi_data_index"],
             )
 
-    gsa_items: Dict[str, GSAItem] = attr.ib(validator=attr.validators.instance_of(Dict))
+    items: Dict[str, GSAItem] = attr.ib(validator=attr.validators.instance_of(Dict))
```

Comment:
:+1: 

We are already on the "Global SA" context because of the class, no need for the attribute to carry that `gsa` prefix.